utils/http.go

```diff
@@ -27,7 +27,16 @@ type rootHandler struct {
 	actions []string
 	context context.Context
 	trust   signed.CryptoService
-	//cachePool redis.Pool
+}
+
+func parseImageName(r *http.Request) string {
+	vars := mux.Vars(r)
```

Comment:
Phew! I thought in my med induced malaise I may have done something really daft :-P